projects/storefrontapp/src/app/spartacus/features/opf/opf-feature.module.ts

```diff
@@ -122,6 +124,15 @@ if (environment.b2b) {
         commerceCloudPublicKey: 'ab4RhYGZ+w5B0SALMPOPlepWk/kmDQjTy2FU5hrQoFg=',
       },
     }),
+    provideConfig(<OpfQuickBuyConfig>{
+      providers: [
+        {
+          googlePay: {
+            resourceUrl: 'https://pay.google.com/gp/p/js/pay.js',
```

Comment:
Should we also update an installation script to add this url together with schematics?